services/QuillLMS/app/lib/rule_feedback_history.rb

```diff
@@ -68,6 +68,7 @@ def self.format_sql_results(relations)
                 api_name: r.rule_type,
                 rule_order: r.rule_suborder,
                 first_feedback: r.feedbacks.order(:order).first&.text || '',
+                second_feedback: r.feedbacks.order(:order).last&.text || '',
```

Comment:
What about `r.feedbacks.order(:order).second&.text || ''`, otherwise if `r.feedbacks` has only one element, `first` and `last` will return the same element.

Reply:
I like `second` here, too.  Also, while no one has ever done it, I'm pretty sure someone could create a "third" layer, and using `last` would pull that one.

Reply:
ahh ok I didn't know you could use `second`, thanks for pointing that out @brendanshean !